### PR TITLE
Ubuntu-based docker image for Resolwe

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

File: resolwe/toolkit/docker_images/base/README.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
 
-It is based on [`docker.io/fedora:25`](https://hub.docker.com/_/fedora/).
-
-The only thing that is added are [Resolwe Runtime Utilities](
+This images include [Resolwe Runtime Utilities](
 http://resolwe-runtime-utils.readthedocs.io).
 
-It is used as the base image for all other Resolwe toolkit Docker images.
+
+The `base` image is based on [`docker.io/fedora:25`](
```
Comment (dblenkus) [inline resolwe/toolkit/docker_images/base/README.md]: It is based on Fedora 26.

**Comment by mstajdohar** [comment]
> What do you think about renaming Dockerfile.base to Dockerfile.centos-base to be consistent with Dockerfile.ubuntu-base?

Do you mean Dockerfile.fedora-base?

**Comment by dblenkus** [comment]
Yes :)

**Comment by mstajdohar** [comment]
Changed

**Comment by mstajdohar** [comment]
I don't know about that what's `:latest` then?

**Comment by kostko** [comment]
> I don't know about that. What's :latest then?

There is no need to have `:latest`?

**Comment by mstajdohar** [comment]
Please review again.

File: MANIFEST.in [hunk 2]
```diff
@@ -22,6 +22,6 @@ recursive-include resolwe/permissions/fixtures *.yaml readme.txt
 recursive-include resolwe/toolkit/tests *.py
 recursive-include resolwe/toolkit/tests/files *
 # include Dockerfiles and files needed to build Docker images
-recursive-include resolwe/toolkit/docker_images Dockerfile README.md
+recursive-include resolwe/toolkit/docker_images Dockerfile* README.md
```
Comment (dblenkus) [inline MANIFEST.in]: Change to `Dockerfile.*`

**Pull request merged**